src/vcs/git.rs

```diff
@@ -445,9 +109,10 @@ impl<'a> Browser<'a> {
     /// # }
     /// ```
     pub fn new(repository: &'a Repository) -> Result<Self, Error> {
+        let repository = repository.as_ref();
```

Comment:
We also need a constructor which takes `RepositoryRef`.

Reply:
I think I'll make `new` take `RepositoryRef` and change the call-sites.

Reply:
Or... Implement `AsRef` :thinking:

Reply:
Used the `Into` trick instead :ok_hand: